system_tests/test_windows_bootstrap.py

```diff
@@ -213,31 +278,22 @@ class TestWindowsOfflineBootstrap(TestWindowsBase, TestOfflineCliPackage):
     def test_offline_windows_cli_package(self):
         self._test_cli_package()
 
-    def _update_hosts_file(self, resolution):
+    def update_hosts_file(self, resolution):
         cmd = 'ac -Encoding UTF8  ' \
               'C:\Windows\system32\drivers\etc\hosts "{0}"'.format(resolution)
         self._execute_command(cmd)
 
-    def _prepare_cli(self):
-        with self.dns():
-            wget_cmd = """
-$client = New-Object System.Net.WebClient
-$url = "{0}"
-$file = "{1}"
-$client.DownloadFile($url, $file)""".format(self.cli_package_url,
-                                            CLI_PACKAGE_EXE)
-            self.logger.info(
-                'Downloading Windows CLI package from: {0}'.format(
-                    self.cli_package_url))
-        self._execute_command(wget_cmd)
+    def prepare_cli(self):
+        self.logger.info('Downloading Windows CLI package from: {0}'
+                         .format(self.cli_package_url))
+        self._get_resource(self.cli_package_url, CLI_PACKAGE_EXE)
 
         return CLI_PACKAGE_EXE
 
-    def _install_cli(self, package_name):
+    def install_cli(self, package_name):
```

Comment:
Actually, the online windows test has an install_cli method, and so does the offline. windowsbase doesn't have it, so it seems fine